resources/todomvc/architecture-examples/vue/src/components/TodosComponent.vue

```diff
@@ -17,36 +13,38 @@
                     :class="{ completed: todo.completed, editing: todo === editing }">
                     <div class="view">
                         <input type="checkbox" v-model="todo.completed" class="toggle">
-                        <label @dblclick="editTodo(todo)">{{ todo.title }}</label>
+                        <label @dblclick="startEdit(todo)">{{ todo.title }}</label>
                         <button class="destroy" @click.prevent="deleteTodo(todo)"></button>
                     </div>
-                    <input type="text" class="edit" v-model="todo.title" @keyup.enter="doneEdit" @blur="doneEdit"
-                        v-todoFocus="todo === editing" />
+                    <input type="text" class="edit" v-model="todo.title" @keyup.enter="finishEdit" @blur="finishEdit" v-todoFocus="todo === editing" />
                 </li>
             </ul>
         </section>
-        <footer class="footer" v-show="todos.length > 0">
-            <span class="todo-count">
-                <strong>{{ remaining }}</strong> {{ pluralizedWord }} left
-            </span>
-            <ul class="filters">
-                <li><router-link to="/" :class="{ selected: route == 'all' }">All</router-link></li>
-                <li><router-link to="/active" :class="{ selected: route == 'active' }">Active</router-link></li>
-                <li><router-link to="/completed" :class="{ selected: route == 'completed' }">Completed</router-link></li>
-            </ul>
-            <button class="clear-completed" v-show="completed" @click.prevent="deleteCompleted">Clear Completed</button>
-        </footer>
+        <TodoFooter :todos="todos" @delete-completed="deleteCompleted" :remaining="activeTodos.length" :completed="completedTodos.length" :route="route"/>
     </section>
 </template>
 
 <script>
 import { nextTick } from 'vue';
+import TodoHeader from './TodoHeader.vue';
+import TodoFooter from './TodoFooter.vue';
+
+const uuid = () => crypto.randomUUID();
```

Comment:
I noticed recently that this isn't working when accessing the app using http on the network, for example when testing from Android to a local dev server.
What do you think of using a simple locally incremented var?

(this could be a change accross the other implementations in an upcoming PR)

Reply:
oh interesting - changed it to the other uuid() method that we are using. 
I'll open a separate pr that'll remove the other usages throughout the todomvc apps.